calculator-gui.py

```diff
@@ -0,0 +1,64 @@
+from tkinter import *
+
+window = Tk()
+window.title('Simple Calculator')
+
+f_num = 0
+
+def button_click(number):
+    current = entry1.get()
+    entry1.delete(0, END)
+    entry1.insert(0, str(current)+str(number))
+def clear():
+    entry1.delete(0, END)
+
+def add_num():
+    num = entry1.get()
+    global f_num
+    f_num = num
+    entry1.delete(0, END)
+
+def get_sum():
```

Comment:
Add other operators also like subtraction, multiplication and division.
Also, get your code in Class format!